pkg/pool-hooks/contracts/hooks-quantamm/HyperSurgeHook.sol

```diff
@@ -142,6 +213,33 @@ contract HyperSurgeHook is BaseHooks, VaultGuard, SingletonAuthentication, Versi
         _setTokenPriceConfigIndex(pool, tokenIndex, hlPairIdx, hlTokenIdx, details);
     }
 
+    struct SetBatchConfigs {
+        TokenPriceCfg tempCfg;
+        uint256 i;
+    }
+
+    /// @notice Batch version (indices).
```

Comment:
NatSpec nit... our codebase only uses /// for one-liners, and /** */ for multi-line. Your codebase seems to be inconsistent here. This file's mostly ///-style, but other files have use /** */ or mix them.